### PR TITLE
demisto/etc#24229: Adding QueryResult output to indicate whether the …

### PR DESCRIPTION
**Pull request by hod-alpert**
…query found a result

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24229

## Description
added output to context path under Domain.Whois.QueryResult to better show if the query found a result

## Minimum version of Demisto
- [x] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by kirbles19** [comment]
@hod-alpert Done.

**Comment by hod-alpert** [comment]
Tests are failing for non-related issue [here](https://github.com/demisto/etc/issues/24296)

**Comment by ShahafBenYakir** [comment]
Why am I assigned here?

File: Packs/Whois/Integrations/Whois/Whois.py [hunk 1]
```diff
@@ -8273,6 +8273,7 @@ def create_outputs(whois_result, domain, query=None):
     ec = {'Name': domain}
     standard_ec = {}  # type:dict
     standard_ec['WHOIS'] = {}
+    standard_ec['QueryResult'] = 'NOT FOUND' not in str(whois_result.get('raw', 'NOT FOUND'))
```
Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois.py:8276]: the issue says the output should be `Not Found` or `Found`, not a boolean

Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois.py:8276]: are you sure that with this implementation this goes under `Domain.Whois` and not `Domain` ?

File: Packs/Whois/Integrations/Whois/Whois_test.py [hunk 2]
```diff
@@ -64,3 +66,35 @@ def cleanup():
...
+         'creation_date': [datetime.datetime(1997, 9, 15, 0, 0)], 'id': ['2138514_DOMAIN_COM-VRSN']},
+        'google.com',
+        True
+    )
+
+]
+
+
+@pytest.mark.parametrize('whois_result, domain, expected', TEST_QUERY_RESULT_INPUT)
+def test_query_result(whois_result, domain, expected):
```
Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois_test.py:97]: lets add the following test cases: 
 - `Found`
 - `raw` is empty
 - `raw` is not a list, but a dict or str

File: Packs/Whois/Integrations/Whois/Whois.py (hunk 1 above)
Comment (hod-alpert) [reply Packs/Whois/Integrations/Whois/Whois.py:8276]: Spoke to @David-BMS about it and agreed, I'll write it in the issue

Comment (hod-alpert) [reply Packs/Whois/Integrations/Whois/Whois.py:8276]: Right

**Pull request merged**